.env.test

```diff
@@ -0,0 +1,10 @@
+KERNEL_CLASS=App\Kernel
```

Comment:
Missing `APP_SECRET` and `SYMFONY_DEPRECATIONS_HELPER` as suggested here: https://github.com/symfony/recipes/blob/master/phpunit/phpunit/4.7/.env.test